tests/namespace/namespace_test.go

```diff
@@ -0,0 +1,304 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 Authors of KubeArmor
+
+package namespace
+
+import (
+	"fmt"
+	"time"
+
+	. "github.com/kubearmor/KubeArmor/tests/util"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = BeforeSuite(func() {
+	err := K8sApply([]string{"res/ubuntu-deployment.yaml"})
+	Expect(err).To(BeNil())
+	KspDeleteAll()
+})
+
+var _ = AfterSuite(func() {
+	err := K8sDelete([]string{"res/ubuntu-deployment.yaml"})
+	Expect(err).To(BeNil())
+	KspDeleteAll()
+})
+
+func getUbuntuPod(name string, ant string) string {
+        pods, err := K8sGetPods(name, "nptest", []string{ant}, 60)
+        Expect(err).To(BeNil())
+        Expect(len(pods)).To(Equal(1))
+        return pods[0]
+}
+
+
+var _ = Describe("Namespace", func() {
+	var ubuntu string
+
+	BeforeEach(func() {
+		ubuntu = getUbuntuPod("ubuntu-1-deployment-","container.apparmor.security.beta.kubernetes.io/ubuntu-1-container: localhost/kubearmor-nptest-ubuntu-1-deployment-ubuntu-1-container")
+	})
+
+	AfterEach(func() {
+		KarmorLogStop()
```

Comment:
Need to delete annotations in `AfterEach()` handling.
Imagine if any of the tests setting the annotations fails midway then all the subsequent tests might fail since the annotation will linger.